godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -280,11 +277,6 @@ public void onPurchasesUpdated(final BillingResult billingResult, @Nullable fina
 		}
 	}
 
-	@Override
-	public void onPriceChangeConfirmationResult(BillingResult billingResult) {
-		emitSignal("price_change_acknowledged", billingResult.getResponseCode());
-	}
```

Comment:
`onPriceChangeConfirmationResult` is a method from `PriceChangeConfirmationListener`, which is [deprecated](https://developer.android.com/reference/com/android/billingclient/api/PriceChangeConfirmationListener)